src/matlab2tikz.m

```diff
@@ -4491,14 +4525,19 @@ function warnFacetedInterp(m2t)
     pgfplotsColormap = sprintf('colormap={%s}{[1%s] %s}',name, unit, join(m2t, colSpecs, '; '));
 end
 % ==============================================================================
-function fontStyle = getFontStyle(m2t, handle)
+function [m2t, fontStyle] = getFontStyle(m2t, handle)
```

Comment:
What i do not like here is the way we handle fontStyle. Would it be possible to use the opts array we have here too? That would make the code less special.